ansible/roles/common/tasks/packages_el8.yml

```diff
@@ -0,0 +1,14 @@
+#vim: set ft=ansible:
+---
+######################### Install Basic Packages
+- name: install common packages for RHEL 8
+  yum:
```

Comment:
Does `yum` work ?

I thought we would have to switch to `package` and let ansible discover the package manager.

Reply:
Yum works fine, both ansible module and command line even though it is dnf behind the scenese. We really don't do anything other than RHEL so I'm inclined to leave it unless you think the `package` module is better than it used to be.

Reply:
Since this is a common role, literally, I am leaving this since all of the new functionality has been folded into the existing config. I don't want to risk breaking something right now.